src/core_plugins/kibana/public/management/sections/indices/_edit.js

```diff
@@ -57,6 +57,13 @@ uiModules.get('apps/management')
     $scope.editSections = Private(IndicesEditSectionsProvider)($scope.indexPattern);
   });
 
+  $scope.indexedFieldTypes = ['string', 'boolean', 'number', 'date', 'geo_point', 'ip'];
```

Comment:
Can you change it so that these arrays are populated dynamically from the values in the index pattern? When you do, make sure that they stay up to date. 
 - When the user adds a scripted field with a previously unused type, the dropdown updates accordingly
 - When the user adds a field to their index with a previously unused type and clicks on the refresh fields button